<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultadosForm</class>
 <widget class="QWidget" name="ResultadosForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>283</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="titleLabel">
        <property name="text">
         <string>Resultados</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="line">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelPreguntasCorrectas">
        <property name="text">
         <string>Preguntas</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelTotalPreguntas">
        <property name="text">
         <string>Correctas</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelScore">
        <property name="text">
         <string>Score</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
